examples/tv-casting-app/tv-casting-common/core/CastingPlayer.cpp

```diff
@@ -210,6 +214,33 @@ void CastingPlayer::ContinueConnecting()
     }
 }
 
+void CastingPlayer::StopConnecting()
+{
+    ChipLogProgress(AppServer, "CastingPlayer::StopConnecting()");
+    CHIP_ERROR err = CHIP_NO_ERROR;
+    mIdOptions.resetState();
+    mIdOptions.mCancelPasscode = true;
+
+    ChipLogProgress(AppServer, "CastingPlayer::StopConnecting() calling SendUserDirectedCommissioningRequest()");
```

Comment:
nit: clarification

```suggestion
    ChipLogProgress(AppServer, "CastingPlayer::StopConnecting() calling SendUserDirectedCommissioningRequest() to indicated canceled passcode entry");
```

Reply:
"to indicate user canceled passcode entry"